src/COVID19/simulation.py

```diff
@@ -135,26 +143,24 @@ def steps(self, n_steps):
             # Save the state of the model
             self.collect_results(next_state, next_action)
             
-            # If at the end_time of the model, restart a new simulation
-            if self.timestep >= self.end_time:
-                self.simulation_number += 1
-                self.start_simulation()
-
-            else:
+            if self.timestep < self.end_time:
                 self.current_state = next_state
                 self.current_action = next_action
                 self.timestep += 1
+            else: # if at the end_time of the model then exit
```

Comment:
**NB**: this actually changes the functionality from how this was before. Before if you specified a number of steps that would take you past the `end_time` it would automatically start a new simulation. However in my view that is unexpected behaviour, it seems more natural to just end the simulation when you reach the end time. If you can still easily start a new simulation by explicitly calling `steps()` again.

This is just my opinion on expected functionality, I'm happy to revert this if desirable.